gramine_ratls_verify.go

```diff
@@ -104,58 +104,91 @@ import "C"
 import (
 	"crypto/x509"
 	"encoding/pem"
-	"errors"
 	"fmt"
+	"log"
+	"os"
 	"unsafe"
-
-	"github.com/sirupsen/logrus"
 )
 
+var Debug bool
 var ra_tls_verify_callback_der_f unsafe.Pointer
+var ra_tls_attest_create_key_and_crt_der_callback_f unsafe.Pointer
 
-// Imports Gramine RA-TLS libraries required to register the callbacks for Quote verification
-// and enclave measurement arguments
 func init() {
-	logrus.SetFormatter(&logrus.TextFormatter{
-		DisableTimestamp: true,
-	})
-	logrus.SetLevel(logrus.DebugLevel)
+	Debug = false
+	if os.Getenv("DEBUG") == "1" {
+		Debug = true
+	}
```

Comment:
```suggestion
	Debug = os.Getenv("DEBUG") == "1"
```

Reply:
is there an advantage of doing this in init, rather than in line 113?

generally speaking, using init is a bad pattern that's better avoided unless explicitly needed. the go authors consider it adding `init` as one of their mistakes ^^

Reply:
> generally speaking, using init is a bad pattern that's better avoided unless explicitly needed. 
Noted. Will include it at L113.